alchemist/alchemist-incarnation-protelis/build.gradle

```diff
@@ -9,10 +9,20 @@ dependencies {
         project(':alchemist-engine'),
         project(':alchemist-loading'),
     )
+    compile 'org.eclipse.xtend:org.eclipse.xtend.lib:2.10.0'
```

Comment:
Why is this? Hardcoded version? Why are files in the protelis incarnation changed?

Reply:
Because Intellij IDEA Xtend plugin is a little buggy...
When builing on the IDE, the plugin adds it automatically to whatever module it founds having xtend files.
I probably forgot to remove it from alchemist-incarnation-protelis module manually.